test/integration/testExecute.js

```diff
@@ -177,166 +178,223 @@ describe('Execute test - variant', function ()
               // Upload sample file
               testUtil.executeCmd(connection, putVariant, callback);
             },
-            function (callback)
-            {
+            function (callback) {
               // Load sample file
               testUtil.executeCmd(connection, copyIntoVariant, callback);
             },
-            function (callback)
-            {
+            function (callback) {
               // Select values from table
               connection.execute({
                 sqlText: selectVariant,
-                complete: function (err, stmt, rows)
-                {
+                complete: function (err, stmt, rows) {
                   var stream = stmt.streamRows();
-                  stream.on('error', function (err)
-                  {
+                  stream.on('error', function (err) {
                     done(err);
                   });
-                  stream.on('data', function (row)
-                  {
+                  // stream.on('data', rowAsserts);
+
+                  stream.on('data', function (row) {
                     // Check the column, header, and value is correct
-                    if (testCase.type == 'XML')
-                    {
-                      assert.strictEqual(row[TEST_COL][TEST_HEADER], TEST_XML_VAL);
-                    }
-                    else if (testCase.type == 'JSON')
-                    {
-                      assert.strictEqual(row[TEST_COL][TEST_HEADER], TEST_JSON_VAL);
-                    }
+                    rowAsserts(testCase, row);
                   });
-                  stream.on('end', function ()
-                  {
+
+
+                  stream.on('end', function () {
                     callback();
                   });
                 }
               });
             },
-            function (callback)
-            {
-              // Drop stage
-              testUtil.executeCmd(connection, dropStageVariant, callback);
-            },
-            function (callback)
-            {
-              // Drop table
-              testUtil.executeCmd(connection, dropTableVariant, callback);
-            },
-            function (callback)
-            {
+            function (callback) {
               // Delete temp files
               fs.closeSync(sampleTempFile.fd);
               fs.unlinkSync(sampleTempFile.name);
               callback();
             }],
           done
         );
-      };
+      }
+      ;
     };
   };
 
-  for (var index = 0; index < testCases.length; index++)
-  {
-    var testCase = testCases[index];
-    it(testCase.name, createItCallback(testCase));
-  }
+  before(async () => {
+    let createFileFormatVariant = `CREATE OR REPLACE FILE FORMAT ${TEST_VARIANT_FORMAT} TYPE = XML`;
+    connection = testUtil.createConnection();
+    await testUtil.connectAsync(connection);
+    await testUtil.executeCmdAsync(connection, createTableVariant);
+  });
+
+  after(async () => {
+    // Drop table
```

Comment:
do we need such comments for such simple cleanup?

Reply:
maybe not -> removed